index.html

```diff
@@ -452,8 +452,12 @@ <h3 id="sec-processing-model"><span class="secno">4.4 </span>Processing Model</h
 			</p>
 		</li>
 </ol>
+<p>At certain points in the algorithms below it is said that a
+<dfn id="document-is-visible">document is visible</dfn> or a <dfn id="document-is-hidden">document is hidden</dfn>. These are
+hooks for <a href="#other-applicable-specifications">other applicable specifications</a> to add additional steps depending on the state of the <a href="#dom-document-hidden"><code>hidden</code></a> attribute.
+</p>

```

Comment:
This is redundant. Having these hooks here doesn't help. Hooks need to be to the actual definitions.